crates/smart-module-development-kit/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "smdk"
+name = "smart-module-development-kit"
```

Comment:
Let's change to `smartmodule-development-kit`.   
We should change other smarmodule crates as well.